kratos/geometries/quadrilateral_2d_8.h

```diff
@@ -1539,16 +1509,20 @@ template<class TPointType> class Quadrilateral2D8
     {
```

Comment:
At the end I keep the static as it is, not calling the methods (it can not be compiled anyway)